torch/testing/_internal/common_methods_invocations.py

```diff
@@ -864,7 +871,7 @@ def reference_sigmoid(x):
 
 # Common operator groupings
 unary_ufuncs = [op for op in op_db if isinstance(op, UnaryUfuncInfo)]
-spectral_funcs = [op for op in op_db if isinstance(op, SpectralFuncInfo)]
+sparse_unary_ufuncs = [op for op in op_db if isinstance(op, UnaryUfuncInfo) and op.sparse_supported is True]
```

Comment:
@hlahkar please revert and keep `spectral_funcs = [op for op in op_db if isinstance(op, SpectralFuncInfo)]`. I think we can reuse `unary_ufuncs` for sparse tests without creating a specific list as you suggested `sparse_unary_ufuncs`. And then you can code
```python
class TestSparseUnaryUfuncs(TestCase):
    exact_dtype = True

    @ops([op for op in unary_ufuncs if op.supports_sparse])
```

Reply:
@vfdev-5 This was done in response to this comment https://github.com/pytorch/pytorch/pull/48488#discussion_r539124174. Do let me know if I need to revert back

Reply:
@hlahkar oh, sorry I haven't seen Mike's comment there. I have no strong opinion about introducing or not such list, so, OK let's keep `sparse_unary_ufuncs`.
However, `spectral_funcs` should be present.

Reply:
@vfdev-5 Sorry missed that at the time of merging, will update the same